build.zig.zon

```diff
@@ -1,5 +1,16 @@
 .{
-  .name = "bytebox",
-  .version = "0.0.1",
-  .dependencies = .{},
+    .name = "bytebox",
+    .version = "0.0.1",
+    .minimum_zig_version = "0.12.0",
+    .paths = .{
+        "src",
+        "test/main.zig",
```

Comment:
I think this has changed since the `memory64` merge to be `test/wasm/main.zig`. Also we'd need to add:
* `test/wasi/main.zig`
* test/mem64/main.zig`

Reply:
Done.